examples/text-generation/run_generation.py

```diff
@@ -223,12 +238,22 @@ def main():
         logger.info(f"Args: {args}")
         logger.info(f"device: {args.device}, n_hpu: {world_size}, bf16: {use_deepspeed or args.bf16}")
 
+    bad_words_ids = None
+    force_words_ids = None
+    if args.bad_words_ids is not None:
+        bad_words_ids = [tokenizer.encode(bad_word) for bad_word in args.bad_words_ids]
```

Comment:
```suggestion
        bad_words_ids = [tokenizer.encode(bad_word, add_special_tokens=False) for bad_word in args.bad_words_ids]
```
As suggested here: https://huggingface.co/docs/transformers/main/en/internal/generation_utils#transformers.NoBadWordsLogitsProcessor